src/common/transformations/tests/op_conversions/convert_interpolate11_downgrade_test.cpp

```diff
@@ -71,16 +72,29 @@ std::shared_ptr<ov::Model> create_v4_model(const bool with_axes,
     if (shape_calc_mode == ov::opset4::Interpolate::ShapeCalcMode::SCALES) {
         scales = std::make_shared<ov::opset4::Parameter>(ov::element::f32, ov::Shape{num_scales_or_sizes});
         model_params.push_back(std::dynamic_pointer_cast<ov::opset4::Parameter>(scales));
-        output_shape = ov::opset4::Constant::create(ov::element::i32, ov::Shape{1}, {1});
-
+        output_shape = ov::opset4::Constant::create(ov::element::i32, ov::Shape{}, {1});
+        if (with_axes) {
+            output_shape =
+                std::make_shared<ov::opset4::Broadcast>(output_shape, std::make_shared<ov::opset4::ShapeOf>(axes));
+        } else {
+            output_shape = std::make_shared<ov::opset4::Broadcast>(
+                output_shape,
+                std::make_shared<ov::opset4::ShapeOf>(std::make_shared<ov::opset4::ShapeOf>(input)));
+        }
```

Comment:
To test the expected final Constant value, there can be additional test case with ConstantFolding pass:
```
    manager.register_pass<ov::pass::ConvertInterpolate11ToInterpolate4>();
    manager.register_pass<ov::pass::ConstantFolding>();
```